code/parachain/runtime/common/src/lib.rs

```diff
@@ -135,7 +135,9 @@ mod constants {
 	pub const NORMAL_DISPATCH_RATIO: Perbill = Perbill::from_percent(75);
 
 	/// We allow for 2 seconds of compute with a 6 second average block time.
-	pub const MAXIMUM_BLOCK_WEIGHT: Weight = WEIGHT_PER_SECOND.saturating_div(2);
+	pub const MAXIMUM_BLOCK_WEIGHT: Weight = WEIGHT_PER_SECOND
+		.saturating_div(2)
+		.set_proof_size(polkadot_primitives::v2::MAX_POV_SIZE as u64);
```

Comment:
Wonder what this does

Reply:
Weight V2 introduce a new `proof_size` field in weights, this just define it's max size as being the max proof of validity size we submit to the relay